.gitignore

```diff
@@ -40,6 +40,7 @@ cmd/query/query-*
 cmd/tracegen/tracegen
 cmd/tracegen/tracegen-*
 crossdock/crossdock-*
+internal/storage/v2/elasticsearch/tracestore/fixtures/actual_*
```

Comment:
No, you should create .gitignore in the fixtures directory, it's a local matter, no reason to manage it at the root level.